### PR TITLE
improves and adds onto tt_descs

### PR DESCRIPTION
**Pull request by elgeonmb**
previously: a best-attempt at binomial nomenclature for earth species (though often making questionable choice of species and using subspecies names in place of species names). Memetastic pseudolatin names for anything else.

currently: a trinomial system using biosphere of origin in addition to genus and species name. Earth species have improved identifications courtesy of belsima, who is a huge fucking nerd. Alien species have vaguely scientific names in any case where they might be regularly encountered. Rare and adminspawn-only mobs get funny names because I'm not totally joyless.

(For reference:

E = Earth
J = Jade
S = Sif
N = Nisp
A = Aetolus
X = Artificially created
U = Unknown origin)

### CODE REVIEW

**Comment by elgeonmb** [comment]
it compiles, fuck you travis

**Comment by Atermonera** [comment]
Active edges again, same issue as your last or, not your fault again

File: code/modules/mob/living/simple_animal/animals/bat.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /mob/living/simple_animal/hostile/scarybat
 	name = "space bats"
 	desc = "A swarm of cute little blood sucking bats that looks pretty upset."
-	tt_desc = "Desmodus rotundus" //Common vampire bat
+	tt_desc = "N Bestia gregaria" //Common vampire bat
```
Comment (Belsima) [inline code/modules/mob/living/simple_animal/animals/bat.dm]: comment is now inaccurate this is very much not the common vampire bat

File: code/modules/mob/living/simple_animal/animals/cat.dm [hunk 2]
```diff
@@ -156,7 +156,7 @@
 /mob/living/simple_animal/cat/fluff/Runtime
 	name = "Runtime"
 	desc = "Her fur has the look and feel of velvet, and her tail quivers occasionally."
-	tt_desc = "Felis medicalis"
+	tt_desc = "E Felis catus medicalis"
```
Comment (Belsima) [inline code/modules/mob/living/simple_animal/animals/cat.dm]: shouldn't this be silvestris medicalis or?

File: code/modules/mob/living/simple_animal/animals/farm_animals.dm [hunk 3]
```diff
@@ -155,7 +155,7 @@
 /mob/living/simple_animal/chick
 	name = "\improper chick"
 	desc = "Adorable! They make such a racket though."
-	tt_desc = "Gallus domesticus"
+	tt_desc = "E Gallus gallus domesticus"
```
Comment (Belsima) [inline code/modules/mob/living/simple_animal/animals/farm_animals.dm]: the 'domesticus' subspecies is generally less recognized and used less commonly each year

File: code/modules/mob/living/simple_animal/animals/farm_animals.dm [hunk 4]
```diff
@@ -206,7 +206,7 @@ var/global/chicken_count = 0
 /mob/living/simple_animal/chicken
 	name = "\improper chicken"
 	desc = "Hopefully the eggs are good this season."
-	tt_desc = "Gallus domesticus"
+	tt_desc = "E Gallus gallus domesticus"
```
Comment (Belsima) [inline code/modules/mob/living/simple_animal/animals/farm_animals.dm]: ditto with above

File: code/modules/mob/living/simple_animal/animals/fish.dm [hunk 5]
```diff
@@ -39,24 +39,28 @@
...
 /mob/living/simple_animal/fish/bass
 	name = "bass"
+	tt_desc = "E Micropterus notius"
 	icon_state = "bass-swim"
 	icon_living = "bass-swim"
 	icon_dead = "bass-dead"
 
 /mob/living/simple_animal/fish/trout
 	name = "trout"
+	tt_desc = "E salmo trutta"
```
Comment (Belsima) [inline code/modules/mob/living/simple_animal/animals/fish.dm]: should capitalize Salmo

File: code/modules/mob/living/simple_animal/humanoids/clown.dm [hunk 6]
```diff
@@ -1,6 +1,7 @@
 /mob/living/simple_animal/hostile/clown
 	name = "clown"
 	desc = "A denizen of clown planet"
+	tt_desc = "E Homo sapiens corydon" //this is an actual clown, as opposed to someone dressed up as one
```
Comment (Belsima) [inline code/modules/mob/living/simple_animal/humanoids/clown.dm]: Wouldn't this be Homo sapiens scurra?

**Comment by Belsima** [comment]
i am a nerd

File: code/modules/mob/living/simple_animal/animals/bat.dm (hunk 1 above)
Comment (MisterLayne) [reply code/modules/mob/living/simple_animal/animals/bat.dm]: What's up with the random letters at the beginning? It's also hard to tell when the animals are actually being given their real scientific names or names or not..

Comment (Atermonera) [reply code/modules/mob/living/simple_animal/animals/bat.dm]: It's a planetary indicator, indicating biosphere of origin, or if they're artificial.
E = Earth
N = Nisp
A = Artificial
There were a few others but I can't seem to find the enumeration

File: code/modules/mob/living/simple_animal/humanoids/clown.dm (hunk 6 above)
Comment (elgeonmb) [reply code/modules/mob/living/simple_animal/humanoids/clown.dm]: H.s. scurra is technically the "fancy" clown; H.s. corydon is the common clown and I think more appropriate for the in-game clown!

**Comment by elgeonmb** [comment]
As I explained in the post above, all Earth clade (names start with E) animals have a scientifically appropriate name. All xenolife has a made-up name as they are in fact, xenolife.

File: code/modules/mob/living/simple_animal/animals/bat.dm (hunk 1 above)
Comment (Belsima) [reply code/modules/mob/living/simple_animal/animals/bat.dm]: X is artificial, A is Aetolus. It's all in the original PRpost.

File: code/modules/mob/living/simple_animal/humanoids/clown.dm (hunk 6 above)
Comment (Belsima) [reply code/modules/mob/living/simple_animal/humanoids/clown.dm]: ah thanks!

File: code/modules/mob/living/simple_animal/animals/corgi.dm [hunk 7]
```diff
@@ -38,7 +38,6 @@
 	real_name = "Ian"	//Intended to hold the name without altering it.
 	gender = MALE
 	desc = "It's a corgi."
-	tt_desc = "Canis commandus"
```
Comment (Belsima) [inline code/modules/mob/living/simple_animal/animals/corgi.dm]: rest in peace comrade

File: code/modules/mob/living/simple_animal/animals/bear.dm [hunk 8]
```diff
@@ -1,8 +1,8 @@
 //Space bears!
 /mob/living/simple_animal/hostile/bear
 	name = "space bear"
-	desc = "RawrRawr!!"
```
Comment (Belsima) [inline code/modules/mob/living/simple_animal/animals/bear.dm]: RawrRawr!!

File: code/modules/mob/living/simple_animal/animals/giant_spider.dm [hunk 9]
```diff
@@ -8,7 +8,7 @@
 /mob/living/simple_animal/hostile/giant_spider
 	name = "giant spider"
 	desc = "Furry and brown, it makes you shudder to look at it. This one has deep red eyes."
-	tt_desc = "Atrax robustus gigantus"
+	tt_desc = "X Brachypelma phorus"
```
Comment (Arokha) [inline code/modules/mob/living/simple_animal/animals/giant_spider.dm]: Fun police 🚓 Removing a great pun that's scientifically accurate!

Comment (Belsima) [reply code/modules/mob/living/simple_animal/animals/giant_spider.dm]: wow holy shit i do not like funnel web spiders!

**Pull request merged**